hugegraph-core/src/main/java/com/baidu/hugegraph/type/HugeType.java

```diff
@@ -48,8 +48,10 @@
     EDGE_IN(140, "I"),
 
     SECONDARY_INDEX(150, "SI"),
-    RANGE4_INDEX(160, "4I"),
-    RANGE8_INDEX(161, "8I"),
+    RANGE_INT_INDEX(160, "I4I"),
```

Comment:
I4I => II
F4I => FI

SEARCH_INDEX => HI
SHARD_INDEX => DI

Reply:
I think keeping range index names consistent is better.